### ISSUE TITLE
Swift 3 - Cannot call value of non-function type 'module<CountdownLabel>'

### ISSUE THREAD

**Issue by ArturBaghdas** [suzuki-0000/CountdownLabel#19]

**Comment by ianscott313** [comment]
Yeah I get this too.  If you command click into the module from the import statement - 'import CountdownLabel,' it looks like the module is not picking up the init functions declared in CountdownLabel.swift.  Thats as far as I have gotten with this bug

**Comment by mittenimraum** [comment]
+1

**Comment by aloveric** [comment]
+1

**Comment by Huddie** [comment]
+1

**Comment by cowboyrushforth** [comment]
this is fixed in #20

**Comment by a9austin** [comment]
I'm still having this same issue. 

+1

**Comment by jsdu** [comment]
+1 doesn't work still

**Comment by Huddie** [comment]
Last I checked, downloading the project and manual installation works. Pods still had the issue

**Comment by YoavTaieb1** [comment]
+1 still don't work (swift 3)

**Comment by fkawasaki** [comment]
I also checked working for manual install but pod install does not work.

**Comment by YoavTaieb1** [comment]
@fkawasaki Can you explain how to install it manually? Thanks

**Comment by gbergamo** [comment]
I'm still getting this error.
iOS 9+
CountdownLabel v2.0.1

`pod 'CountdownLabel', :git => 'https://github.com/suzuki-0000/CountdownLabel', :branch => 'master'`

`
import CountdownLabel

override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(true)
        
        let countdownLabel = CountdownLabel(frame: CGRect(x: 0, y: 0, width: 100, height: 100), minutes: 30) // you can use NSDate as well
        countdownLabel.start()
    }
`

**Comment by kcyuanGit** [comment]
+ 1

**Comment by kcyuanGit** [comment]
+1

**Comment by stefanopassador** [comment]
+1

**Comment by jihongboo** [comment]
+1

**Comment by bernsrd-systempsrtnets** [comment]
+1

**Comment by hanishassim** [comment]
+1

**Comment by rickyjwhong** [comment]
+1

**Comment by ferusinfo** [comment]
+1

**Comment by tgwagner** [comment]
+1
